packages/remote-config/lib/index.js

```diff
@@ -282,6 +289,45 @@ class FirebaseConfigModule extends FirebaseModule {
     return this._promiseWithConstants(this.native.setDefaultsFromResource(resourceName));
   }
 
+  /**
+   * Registers a listener to changes in the configuration.
+   *
+   * @param listenerOrObserver - function called on config change
+   * @returns {function} unsubscribe listener
+   */
+  onConfigUpdated(listenerOrObserver) {
+    const listener = this._parseListener(listenerOrObserver);
+    const subscription = this.emitter.addListener(
+      this.eventNameForApp('onConfigUpdated'),
+      event => {
+        const { resultType } = event;
+        delete event.resultType;
+
+        if (resultType === 'success') {
+          Promise.resolve().then(() => {
+            listener(event, undefined);
+          });
+          return;
+        }
+
+        Promise.resolve().then(() => {
+          listener(undefined, event);
+        });
+      },
```

Comment:
I think this can be removed now, this is specific to auth logic

Reply:
Done